pkg/config/config.go

```diff
@@ -95,6 +104,11 @@ const (
 	ConfigmapReloaderImgTagSuffix = "4.5.0"
 	ConfigmapReloaderKey          = "prometheus-config-reloader"
 
+	OauthProxyImgRepo      = "quay.io/openshift"
+	OauthProxyImgName      = "origin-oauth-proxy"
+	OauthProxyImgTagSuffix = "4.5.0"
+	OauthProxyKey          = "alertmanager-oauth-proxy"
```

Comment:
it should be `oauth_proxy` which exists in `oc get cm -n open-cluster-management -oyaml mch-image-manifest-2.3.0`

Reply:
good catch! Addressed.
I found that `oauth-proxy` image only exists in `mch-image-manifest` configmap and the not always built and tagged. I need to handle it specially.